app/views/layouts/application.html.erb

```diff
@@ -13,7 +13,7 @@
     <%= yield :head %>
   </head>
 
-  <body>
+  <body style="background-color:#f3f3f3;">
```

Comment:
@cdoyle-temple Could this inline style be changed to a class?

Reply:
Yes there is a style with $background-grey set for the background (main.scss:10). Will remove the inline style.